dockerfiles/local-data-feed/samples/btc-usdt.aggregator.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
For aggregator, aggregator address is required which is expected to change everytime the docker file is up. rather than downloading and modifying I tried to just keep it in the folder